Integrations/integration-VxStream.yml

```diff
@@ -284,6 +288,30 @@ script:
         return res;
     }
 
+    function detonateFile(entryId, environmentId) {
+
+        var hash = submitFile(entryId, environmentId).substring(10);
+
+        var delay = 60000;
```

Comment:
Delay and total wait time should be arguments to the command. You can provide sensible defaults but why hardcode?